packages/jetpack-ai-calypso/src/logo-generator/components/prompt.tsx

```diff
@@ -18,7 +35,9 @@ export const Prompt: React.FC = () => {
 			</div>
 			<div className="jetpack-ai-logo-generator__prompt-query">
 				<textarea placeholder="describe your site or simply ask for a logo specifying some details about it"></textarea>
```

Comment:
I guess this placeholder will need to be wrapped in a translate function, right?

Reply:
Yes. Since we are using a feature branch and it just works on development we just left it as is for now, but we'll definitely need to wrap it.